sig-cloud-provider/annual-report-2023.md

```diff
@@ -12,6 +12,33 @@
    - Governance and leadership changes
 -->
 
+[**Bridget Kromhout and Michael McCune elected as new SIG chairs**](https://groups.google.com/g/kubernetes-sig-cloud-provider/c/BAh7gGZhfvo/m/5eJtVdoIDwAJ)
+
+Previous chairs Andrew Sy Kim and Nick Turner stepped down during 2023, with Andrew staying on as a technical lead and Nick becoming an emeritus lead. Many thanks for their leadership and guidance over the years.
+
+[**[KEP-2395] Update 2395-removing-in-tree-cloud-providers to beta status**](https://github.com/kubernetes/enhancements/pull/4190)
+
+Updating KEP-2395 was the largest task that the SIG accomplished during 2023. During the update
+of this KEP we changed the default behavior for Kubernetes to use external cloud controllers.
+This update could not have been achieved without the support of the wider Kubernetes community as
+there were several crucial changes to the CI and testing infrastructure which needed to be
+coordinated. Thank you to all who helped!
```

Comment:
Do we want to list the thank-yous from the KubeCon talk?

Reply:
could do, i went with something more general but happy to change it.

Reply:
added thank you list from the retro slide from kubecon eu 2024 preso.